### ISSUE TITLE
Dependency Issue: legacy-swc-helpers in @pdfme/generator

### ISSUE THREAD

**Issue by taylorlacort** [pdfme/pdfme#563]
### Describe the bug

There is a security concern with one of the transitive dependencies used in `@pdfme/generator`. The package `legacy-swc-helpers` has been identified as potentially malicious and can pose security risks.

### To Reproduce

1. Install `@pdfme/generator`.
2. Run a security scan using tools like Snyk.
3. Observe the warning regarding `legacy-swc-helpers`.

### Expected behavior

It is important for the project to either remove this dependency or update it to ensure the security and integrity of the projects relying on `@pdfme/generator`.

### Your Environment

```markdown
- pdfme package(@pdfme/generator or @pdfme/ui):
- pdfme version:
- Operating system:
- Node.js version or Browser name & version:
```

### Your Error Log

```shell
For more details about this vulnerability, please refer to the Snyk report: [SNYK-JS-LEGACYSWCHELPERS-7647380](https://security.snyk.io/vuln/SNYK-JS-LEGACYSWCHELPERS-7647380).
```

**Comment by peteward** [comment]
Looks like we need to update fontkit to `^2.0.4`

**Comment by hkup859** [comment]
I believe this is a false positive. It appears there's an odd naming pattern where the legacy-swc-helpers actually resolves to npm:@swc/helpers@=0.4.14 and not the malicious code. The malicious attacker appears to have been trying to use the naming confusion to their advantage, but it appears there isn't actually a concern here.

swc maintainers: https://github.com/swc-project/swc/discussions/9400

Package resolving safely: https://github.com/pdfme/pdfme/blob/538ed5d5f0c669890ec02226b61921f015580da9/package-lock.json#L2513
